packages/shared/src/UI/components/CollectionDetailCard/index.tsx

```diff
@@ -0,0 +1,276 @@
+        fontWeight: 400,
+        display: 'flex',
+        alignItems: 'center',
+    },
+    threeLine: {
+        display: '-webkit-box',
+        '-webkit-line-clamp': 3,
+        height: 60,
+        fontSize: 14,
+        fontWeight: 400,
+        width: '100%',
+        overflow: 'hidden',
+        textOverflow: 'ellipsis',
+        '-webkit-box-orient': 'vertical',
+    },
+    themeColor: {
+        color: theme.palette.maskColor.highlight,
+    },
+    linkLogo: {
+        width: 24,
+        height: 24,
+    },
+    icons: {
+        margin: '16px 0 16px 0',
+        display: 'flex',
+        alignItems: 'center',
+    },
+    traitsBox: {
+        marginTop: 16,
+        gridRowGap: 16,
+        gridColumnGap: 20,
+        display: 'grid',
+        gridTemplateColumns: 'repeat(3, 170px)',
+    },
+    traitItem: {
+        backgroundColor: theme.palette.maskColor.bg,
+        borderRadius: 8,
+        padding: 12,
+    },
+    traitValue: {
+        fontSize: 14,
+        fontWeight: 700,
+        color: theme.palette.maskColor.main,
+    },
+    secondText: {
+        fontSize: 14,
+        fontWeight: 400,
+        color: theme.palette.maskColor.second,
+    },
+}))
+
+const ChainID = {
+    ethereum: ChainId.Mainnet,
+    polygon: ChainId.Matic,
+    bnb: ChainId.BSC,
+}
+
+export enum CollectionType {
+    donations = 'Donations',
+    footprints = 'Footprints',
+    feeds = 'Feeds',
+}
+
+export const CollectionDetailCard = memo<CollectionDetailCardProps>(
+    ({
+        img,
+        open,
+        onClose,
+        title,
+        referenceURL,
+        metadata,
+        description,
+        date,
+        location,
+        relatedURLs = EMPTY_LIST,
+        traits,
+        type,
+        time,
+        tokenAmount,
+        tokenSymbol,
+        hash,
+    }) => {
+        const t = useSharedI18N()
+        const { classes } = useStyles()
+
+        const icons = relatedURLs.map((url) => {
+            let icon: ReactNode = null
+            if (url.includes('etherscan.io')) {
+                icon = <Icons.EtherScan size={24} sx={{ marginRight: '12px' }} />
+            } else if (url.includes('polygonscan.com/tx')) {
+                icon = <Icons.PolygonScan size={24} sx={{ marginRight: '12px' }} />
+            } else if (url.includes('polygonscan.com/token')) {
+                icon = <Icons.PolygonScan size={24} sx={{ marginRight: '12px' }} />
+            } else if (url.includes('opensea.io')) {
+                icon = <Icons.OpenSeaColored size={24} sx={{ marginRight: '12px' }} />
+            } else if (url.includes('gitcoin.co')) {
+                icon = <Icons.Gitcoin size={28} sx={{ marginRight: '12px' }} />
+            }
+            return icon ? (
+                <Link href={url} target="_blank" marginBottom="8px">
+                    {icon}
+                </Link>
+            ) : null
+        })
+
+        const days = differenceInCalendarDays(Date.now(), new Date(time ?? 0))
```

Comment:
Do you need `formatDistanceToNow`?

Reply:
`formatDistanceToNow` can not meet my needs